core/tests/dispatch_integration_tests.rs

```diff
@@ -1060,107 +1060,107 @@ fn remote_forwarding_named() {
-        BigPingerAct::new_preserialised(ponger_named_path1, 12800, BufferConfig::default())
-    });
-    pinf.wait_expect(Duration::from_millis(1000), "Pinger failed to register!");
-    let ponger_named_path2 = ponger_named_path;
-    let (small_pinger1_named, pinf) = small_pinger_system.create_and_register(move || {
-        BigPingerAct::new_preserialised(ponger_named_path2, 10, BufferConfig::default())
-    });
-    pinf.wait_expect(Duration::from_millis(1000), "Pinger failed to register!");
-
-    // Ponger_system blocked for 10 Seconds from this time.
-    ponger_system.start(&ponger_named);
-    big_pinger_system.start(&big_pinger_named);
-
-    // TODO maybe we could do this a bit more reliable?
-    thread::sleep(Duration::from_millis(4000));
-
-    // remote system should be unable to receive any messages as the BigPing is occupying all buffers
-    small_pinger_system.start(&small_pinger1_named);
-
-    // Give the sytem time to fail to establish connection
-    thread::sleep(Duration::from_millis(4000));
-
-    // Start the second Pinger and assert that small_pinger1 hasn't gotten the pong yet.
-    // small_pinger_system.start(&small_pinger2_named);
-    let pingfn = small_pinger_system.stop_notify(&small_pinger1_named);
-    pingfn
-        .wait_timeout(Duration::from_millis(1000))
-        .expect("Pinger never stopped!");
-    small_pinger1_named.on_definition(|c| {
-        assert_eq!(c.count, 0);
-    });
-    small_pinger_system.start(&small_pinger1_named);
-
-    // Shutdown big_pinger_system to make sure it won't continue blocking.
-    // Assert that the big_pinger never got anything as a sanity check.
-    let pingfn = big_pinger_system.stop_notify(&big_pinger_named);
-    pingfn
-        .wait_timeout(Duration::from_millis(1000))
-        .expect("Pinger never stopped!");
-    big_pinger_named.on_definition(|c| {
-        assert_eq!(c.count, 0);
-    });
-    big_pinger_system
-        .shutdown()
-        .expect("Kompact didn't shut down properly");
-
-    // Wait for the big_pinger_system connection to time_out
-    // and let the small_pinger system establish its connection and succeed with its messages
-    thread::sleep(Duration::from_millis(20000));
-
-    // Assert that small_pinger2 has gotten the pongs.
-    let pingfn = small_pinger_system.stop_notify(&small_pinger1_named);
-    pingfn
-        .wait_timeout(Duration::from_millis(1000))
-        .expect("Pinger never stopped!");
-    small_pinger1_named.on_definition(|c| {
-        assert_eq!(c.count, 2 * PING_COUNT);
-    });
-
-    // Shut down the ponger
-    let pongfn = ponger_system.kill_notify(ponger_named);
-    pongfn
-        .wait_timeout(Duration::from_millis(1000))
-        .expect("Ponger never died!");
-
-    ponger_system
-        .shutdown()
-        .expect("Kompact didn't shut down properly");
-    small_pinger_system
-        .shutdown()
-        .expect("Kompact didn't shut down properly");
-}
+// #[test]
```

Comment:
Do you want to merge it with this test commented out? I kinda just did that as testy thingy to see if it helped to run it on GHA...

Reply:
Yeah it's too flaky. I'm gonna replace it with a better test with my next PR.